taprpc/taprootassets.proto

```diff
@@ -668,8 +674,58 @@ message ProofFile {
     string genesis_point = 2;
 }
 
-message ProofVerifyResponse {
+message DecodedProof {
+    // The index depth of the decoded proof, with 0 being the latest proof.
+    int64 proof_at_depth = 1;
+
+    // The total number of proofs contained in the raw proof.
+    int64 number_of_proofs = 2;
+
+    // The asset referenced in the proof.
+    Asset asset = 3;
+
+    // The reveal meta data associated with the proof, if available.
+    AssetMeta meta_reveal = 4;
+
+    // The merkle proof for AnchorTx used to prove its
+    // inclusion within BlockHeader.
+    bytes tx_merkle_proof = 5;
+
+    // The TaprootProof proving the new inclusion of the
+    // resulting asset within AnchorTx.
+    bytes inclusion_proof = 6;
+
+    // The set of TaprootProofs proving the exclusion of
+    // the resulting asset from all other Taproot outputs within AnchorTx.
+    repeated bytes exclusion_proofs = 7;
+
+    // An optional TaprootProof needed if this asset is
+    // the result of a split. SplitRootProof proves inclusion of the root
+    // asset of the split.
+    bytes split_root_proof = 8;
```

Comment:
There is also the `AdditionalInputs` field. But those are whole proof files (with each a list of proofs inside), which would make this too big. But maybe just `num_additional_inputs` as an information and if the user needs to know what's inside those, they can decode them themselves recursively?